swagger-coding-style.md

```diff
@@ -526,7 +526,7 @@ annotation to the file parameter with the value `true`.
 Applications that run in Bluemix can obtain credentials for thier associated services from the `VCAP_SERVICES` environment
 variable.
 
-Specify the `x-vcap-services-name` annotation in the info section of the swagger file with the name of the service
+Specify the `x-vcap-service-name` annotation in the info section of the swagger file with the name of the service
```

Comment:
At some point we should probably do a global search and replace of "swagger" to "API definition" or some equivalent term, but that's not the point of this PR so we can think about that change for the future I guess.